pages/create-issue.tsx

```diff
@@ -61,45 +105,54 @@ export default function PageCreateIssue() {
         <form onSubmit={createIssue}>
           <div className="container">
             <div className="row justify-content-center">
-              <div className="col-md-6">
+              <div className="col-md-10">
                 <div className="content-wrapper mt-up mb-5">
-                  <h4 className="h4 mr-2">Lock</h4>
+                  <h3 className="h3 mr-2 mb-4">Details</h3>
                   <div className="form-group mb-4">
-                    <label className="p-small trans mb-2">Issue title</label>
+                    <label className="p-small mb-2">Issue title</label>
                     <input type="text"
                       className="form-control" placeholder="Your issue title"
                       value={issueTitle}
                       onChange={e => setIssueTitle(e.target.value)}
                       />
+                    <p className="p-small trans my-2">Tip: Try to be as much descriptive as possible</p>
                   </div>
-                  <div className="form-group mb-4">
-                    <label className="p-small trans mb-2">Set $BEPRO ammout</label>
-                    <input min="0" type="number" className="form-control" placeholder="0"
-                      value={issueAmount}
-                      onChange={e => setIssueAmount(parseInt(e.target.value))}/>
+                  <div className="form-group col-md-4 mb-4">
+                    <label className="p-small mb-2">Set $BEPRO value</label>
+                    <div className="input-group">
+                      <NumberFormat min="0" max={`${balance}`} className="form-control" placeholder="0"
```

Comment:
```suggestion
                      <NumberFormat min="0" max={balance} className="form-control" placeholder="0"
```
You can remove the template on `[max]` and set the expression directly.

Reply:
Lack of attention from me, sorry. Done -> dfb16bd